pyannote/audio/core/inference.py

```diff
@@ -552,16 +552,6 @@ def aggregate(
 
         num_chunks, num_frames_per_chunk, num_classes = scores.data.shape
 
-        chunks = scores.sliding_window
-        frames = SlidingWindow(
-            start=chunks.start,
-            duration=frames.duration,
-            step=frames.step,
-        )
```

Comment:
I don't think those should be removed.

Reply:
I can definitely add this back. I'm curious, under what circumstances would chunks.start be different from frames.start? I started looking into the code, but I did not get that far.

Reply:
There might be cases where we only process part of a file so there is no guaranteee that `scores` are aligned with the beginning of the file.